wplms-coauthor-plus.php

```diff
@@ -44,19 +44,24 @@ function wplms_coauthor_plus_instructor($instructor, $id,$r = null){
       foreach($coauthors as $k=>$inst){
         $instructor_id = $inst->ID;
         $displayname = bp_core_get_user_displayname($instructor_id);
-        if(function_exists('vibe_get_option'))
+        if(function_exists('vibe_get_option')) {
           $field = vibe_get_option('instructor_field');
+          if(!isset($field) || $field =='') $field='Speciality';
```

Comment:
Hi @MrVibe,
With the closing bracket moved there will be no $field if the function vibe_get_option is not a defined function.

I'd suggest for this block to have the default set before the vibe function check so if it fails then $field is still set to default.

Something like this;
$field='Speciality';
if(function_exists('vibe_get_option')) {
  $vibe_instructor_field = vibe_get_option('instructor_field');
  if(!empty($vibe_instructor_field)) $field = $vibe_instructor_field; 
}

Hope that makes sense, so the default would be 'Specialty' unless vibe_get_option function exists and the instructor_field option is specified.

Cheers